core/src/main/java/io/undertow/server/BasicSSLSessionInfo.java

```diff
@@ -66,7 +68,8 @@ public BasicSSLSessionInfo(byte[] sessionId, String cypherSuite, String certific
             int i=0;
             for(java.security.cert.Certificate cert : certCol) {
                 this.peerCertificate[i] = cert;
-                this.certificate[i++] = X509Certificate.getInstance(cert.getEncoded());
+//                this.certificate[i++] = X509Certificate.getInstance(cert.getEncoded());
```

Comment:
There is a commented line of code. Forgot to delete

Reply:
Sortof.  Fixed.